src/pytest_inmanta_lsm/plugin.py

```diff
@@ -433,6 +433,17 @@ def remote_orchestrator(
     }
     settings.update(remote_orchestrator_settings)
 
+    # Get server version and check if the server has the patch which safely allows to reset
+    # the agent map
+    # https://github.com/inmanta/inmanta-core/issues/7448
+    orchestrator_version = version.Version(remote_orchestrator_shared.status.version)
+    good_iso7 = orchestrator_version > version.Version("7.1.0.dev")
+    good_iso6 = version.Version("7.dev") > orchestrator_version > version.Version("6.5.0.dev")
```

Comment:
```suggestion
    good_iso6 = version.Version("7") > orchestrator_version > version.Version("6.5.0.dev")
```
`7 < 7.dev`

Reply:
```
>>> version.Version("7") < version.Version("7.dev")
False
```

Reply:
You're absolutely right, not sure what I was thinking.